### PR TITLE
NFTManager generic action template

### PR DESCRIPTION
**Pull request by tspoff**

### CODE REVIEW

File: src/genericPluginRegistry/plugins/NFTManager.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
+{
+    "name": "NFTManager",
+    "addresses": {
+      "main":  ["0x478D17C854446a8718Dc42A8199444f26Ddfe1Ad"],
```
Comment (orenyodfat) [inline src/genericPluginRegistry/plugins/NFTManager.json:4]: there is no such contract on mainnet

**Comment by orenyodfat** [comment]
lint error

File: src/genericPluginRegistry/plugins/NFTManager.json (hunk 1 above)
Comment (tspoff) [reply src/genericPluginRegistry/plugins/NFTManager.json]: Removed 👍

**Comment by orenyodfat** [comment]
any test dao which demonstrate this NFT functionality  ?

**Pull request closed without merging**